### PR TITLE
chore(bot): add desc for bot configs

### PR DESCRIPTION
**Pull request by IvanJobs**
to provide support for https://github.com/OfficeDev/TeamsFx/issues/1238

### CODE REVIEW

File: docs/fx-core/bot-help.md [hunk 1]
```diff
@@ -65,4 +65,23 @@ Toolkit v2 will pop-up a dialog to show the target message endpoint for bot. Use
...
\ No newline at end of file
+After updating message endpoint is done, continue to deploy and try the remote experience.
+
+## Configuration schema for bot
+This section is to describe configuration items in `fx-resource-bot` section of `.fx/env.default.json`.
+
+Config Name | Config Type | Description
+------|------|------
+wayToRegisterBot|string|The way to register bot, one is `create-new`, the other is `reuse-existing`.
+skuName|string|The sku name for azure web app which is used for hosting the bot project on Azure.
```
Comment (eriolchan) [inline docs/fx-core/bot-help.md:76]: can we give some examples of skuName value for user

Comment (eriolchan) [inline docs/fx-core/bot-help.md:76]: can we unify the word of "Azure Web App", I see here in Line 76 is different from that in line 80

**Review by jayzhang — APPROVED** [review]

File: docs/fx-core/bot-help.md (hunk 1 above)
Comment (IvanJobs) [reply docs/fx-core/bot-help.md:76]: sure

Comment (IvanJobs) [reply docs/fx-core/bot-help.md:76]: sure.

**Review by eriolchan — APPROVED** [review]

**Review by YitongFeng-git — APPROVED** [review]

**Pull request merged**